### PR TITLE
Minimum Cost to Connect Two Groups of Points Solution (Leetcode) Added

### PR DESCRIPTION
**Pull request by vivekvardhan2810**
## Related Issue

Fixed : #783 

## Description

I had Added the Solution of problem "Minimum Cost to Connect Two Groups of Points"  under Leetcode Solution > 1500 - 1599> 1595 House Robber

## Type of PR

- [ ] Bug fix
- [x] Feature enhancement
- [x] Documentation update
- [ ] Other (specify): _______________

## Screenshots / Videos (if applicable)

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

**Comment by vivekvardhan2810** [comment]
Hi @Ajay-Dhangar , i have added solution you can review it.

And tell me the suggestions.

File: dsa-solutions/lc-solutions/1500-1599/1595 - Minimum Cost to Connect Two Groups of Points.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+- Bit Manipulation
+- Java
+- Bitmask
+- Matrix
+- Array
+description: "This document provides a solution where we return the minimum cost to connect the two groups."
+---
+## Problem
+
+You are given two groups of points where the first group has size1 points, the second group has size2 points, and size1 >= size2.
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/1500-1599/1595 - Minimum Cost to Connect Two Groups of Points.md:16]: replace `size1 >= size2` to `$size1 >= size2$`

Comment (Ajay-Dhangar) [reply dsa-solutions/lc-solutions/1500-1599/1595 - Minimum Cost to Connect Two Groups of Points.md:16]: and also, `size1` to `$size1$` , and `size2` to `$size2$`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by vivekvardhan2810** [comment]
@Ajay-Dhangar resolved, can you check it and give suggestions

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Comment by vivekvardhan2810** [comment]
Thank you @Ajay-Dhangar for the Suggestions and support.

**Pull request merged**